lib/core/src/display/workspace.rs

```diff
@@ -170,80 +184,76 @@ impl<OnDirty: Clone + Callback0 + 'static> Workspace<OnDirty> {
         gl_Position = position;
     }
 "#,
-            )
-            .unwrap();
-            let frag_shader = webgl::compile_shader(
-                &self.context,
-                webgl::Context::FRAGMENT_SHADER,
-                r#"
-    void main() {
-        gl_FragColor = vec4(1.0, 1.0, 1.0, 1.0);
-    }
-"#,
-            )
-            .unwrap();
-            let program =
-                webgl::link_program(&self.context, &vert_shader, &frag_shader).unwrap();
-
-            let pos_loc = self.context.get_attrib_location(&program, "position");
-            let pos_loc = pos_loc as u32;
-
-            println!("pos_loc: {}", pos_loc);
-
-            let vertices: [f32; 9] = 
-                 [ -1.0, -1.0, 0.0
-                 ,  1.0, -1.0, 0.0
-                 ,  0.0,  1.0, 0.0
-                 ];
-
-            let buffer = self.context.create_buffer().ok_or("failed to create buffer").unwrap();
-            self.context.bind_buffer(webgl::Context::ARRAY_BUFFER, Some(&buffer));
-
-            // Note that `Float32Array::view` is somewhat dangerous (hence the
-            // `unsafe`!). This is creating a raw view into our module's
-            // `WebAssembly.Memory` buffer, but if we allocate more pages for ourself
-            // (aka do a memory allocation in Rust) it'll cause the buffer to change,
-            // causing the `Float32Array` to be invalid.
-            //
-            // As a result, after `Float32Array::view` we have to be very careful not to
-            // do any memory allocations before it's dropped.
-            unsafe {
-                let vert_array = js_sys::Float32Array::view(&vertices);
-
-                self.context.buffer_data_with_array_buffer_view(
-                    webgl::Context::ARRAY_BUFFER,
-                    &vert_array,
-                    webgl::Context::STATIC_DRAW,
+                )
+                .unwrap();
+                let frag_shader = webgl::compile_shader(
+                    &self.context,
+                    webgl::Context::FRAGMENT_SHADER,
+                    r#"
+        void main() {
+            gl_FragColor = vec4(1.0, 1.0, 1.0, 1.0);
+        }
+    "#,
```

Comment:
Do we need these changes (a lot of lines up and down)? I believe they are just formatting of my code which is just going to be completely changed on my next PR, so in order to not have some strange collisions, I would vote for not making these changes now :P

Reply:
It is not even reformatting... Just increase level of indentation, because I wrapped this code in if (to make the text example work.

For sake of merging I can revert it - it will be wrongly indented, but still, this code soon disappears :)